pkg/bucket/s3_test.go

```diff
@@ -45,8 +45,7 @@ var _ = Describe("S3Bucket", func() {
 				AccessKeyID:     "minioadmin",
 				SecretAccessKey: "minioadmin",
 				Source:          "minio default credentials",
-			},
-		}))
+			}}), config.WithRegion("us-west-2"))
```

Comment:
Plase use [`WithRegion()`](https://github.com/cybozu-go/moco/blob/v0.20.2/pkg/bucket/s3.go#L30-L35).
And it's a trivial thing but the default region of MinIO seems to be `us-east-1`.

```suggestion
		cfg, err := config.LoadDefaultConfig(ctx, config.WithCredentialsProvider(credentials.StaticCredentialsProvider{
			Value: aws.Credentials{
				AccessKeyID:     "minioadmin",
				SecretAccessKey: "minioadmin",
				Source:          "minio default credentials",
			}}))
		Expect(err).NotTo(HaveOccurred())
		client := s3.NewFromConfig(cfg,
			WithEndpointURL("http://localhost:9000"),
			WithRegion("us-east-1"), // Please specify here.
			WithPathStyle(),
		)
```

Reply:
@masa213f 
Thank you for your code that I should add.
I will change the code arroding to your recomendation.

Reply:
Please reflect this comment.

Reply:
Sorry, I don't undestand your this reply.
I changed region us-test-1,
 Is it not enough ?

Reply:
Please read suggested code.

Reply:
@masa213f 
I changed suggested code.
Is it right ?

Reply:
@masa213f 
I updated for this